hubblestack/grains/hubble_core.py

```diff
@@ -2376,11 +2372,11 @@ def dns():
     if hubblestack.utils.platform.is_windows() or 'proxyminion' in __opts__:
         return {}
 
-    resolv = salt.utils.dns.parse_resolv()
+    resolv = hubblestack.utils.dns.parse_resolv()
     for key in ('nameservers', 'ip4_nameservers', 'ip6_nameservers',
                 'sortlist'):
         if key in resolv:
-            resolv[key] = [six.text_type(i) for i in resolv[key]]
+            resolv[key] = [str(i) for i in resolv[key]]
```

Comment:
Are you sure str() is right?

I mean, I think you're changing numbers to strings here, but in the worst case, str() doesn't correctly convert bytes to strings.

Reply:
Since we are working on Python-3. Old code's six.text_type is str only.